### PR TITLE
Adds the T-35 Pump shotgun to marines

### PR DESCRIPTION
**Pull request by Kuro-ai**
Credits to Mileo_inc for doing all the mob and stock sprites!!

## About The Pull Request

Adds the (recolored m37) T-35 pump shotgun for marines,buffs buckshot and flechette

## Why It's Good For The Game

More uniformity in lore and looks,buffs to buckshot and flechette so they stop being shit

## Changelog
:cl: Kurp-ai and Mileo_inc
add:Add the T-35 pump shotgun
balance:buff flechette and buckshot
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -5. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

File: code/datums/gamemodes/distress.dm [hunk 1]
```diff
@@ -359,7 +359,7 @@
 						/obj/item/storage/backpack/marine/standard = round(scale * 15),
 						/obj/item/storage/backpack/marine/satchel = round(scale * 15),
 						/obj/item/storage/large_holster/machete/full = round(scale * 10),
-						/obj/item/storage/large_holster/m37 = round(scale * 10),
+						/obj/item/storage/large_holster/t35 = round(scale * 10),
```
Comment (Rohesie) [inline code/datums/gamemodes/distress.dm]: Just remove this.

File: code/modules/projectiles/updated_projectiles/gun_attachables.dm [hunk 2]
```diff
@@ -928,6 +928,17 @@ Defined in conflicts.dm of the #defines folder.
...
+/obj/item/attachable/stock/t35stock
+	name = "\improper T-35 stock"
+	desc = "A non-standard heavy stock for the T-35 Shotgun. Less quick and more cumbersome than the standard issue stakeout, but reduces recoil and improves accuracy. Allegedly makes a pretty good club in a fight too."
+	slot = "stock"
+	wield_delay_mod = 0.4 SECONDS
+	icon_state = "t35stock"
+	accuracy_mod = 0.15
+	recoil_mod = -3
+	scatter_mod = -20
+	movement_acc_penalty_mod = 0.1
```
Comment (Rohesie) [inline code/modules/projectiles/updated_projectiles/gun_attachables.dm]:
```suggestion
```
Let's put this var to rest.

File: code/modules/projectiles/updated_projectiles/guns/shotguns.dm [hunk 3]
```diff
@@ -709,3 +709,43 @@ can cause issues with ammo types getting mixed up during the burst.
 	recoil = 2
 	recoil_unwielded = 4
 	pump_delay = 2
+
+//------------------------------------------------------
+//T-35 Pump shotgun
+/obj/item/weapon/gun/shotgun/pump/t35
+	name = "\improper T-35 Pump shotgun"
```
Comment (Rohesie) [inline code/modules/projectiles/updated_projectiles/guns/shotguns.dm]:
```suggestion
	name = "\improper T-35 pump shotgun"
```

**Pull request merged**